pom.xml

```diff
@@ -12,7 +12,7 @@
   <version>1.36-SNAPSHOT</version>
 
   <properties>
-    <java.level>3</java.level>
+    <java.level>6</java.level>
```

Comment:
so what is the status quo if any? :-)
Java 1.3 or 1.6 as a minimum. I don't mind reverting my changes if needed
Let me know.